pallets/storage-provider/benchmarks/src/mock.rs

```diff
@@ -100,6 +72,7 @@ impl pallet_proofs::Config for Test {
 // Sourced from the Testnet runtime defined in <runtime/src/configs/mod.rs>.
 parameter_types! {
     // Storage Provider Pallet
+    pub const StoragePalletId: PalletId = PalletId(*b"Storagee");
```

Comment:
```suggestion
    pub const StoragePalletId: PalletId = PalletId(*b"Storage");
```
What is this Id even used for, though?